packages/trampoline/README.md

```diff
@@ -0,0 +1,9 @@
+# supertramp
+
+> Sync and async trampolines, for your pleasure
+
+## Wat
+
+From [Wikipedia](https://en.wikipedia.org/wiki/Trampoline_(computing)):
+
+> A trampoline is a loop that iteratively invokes [thunk](https://en.wikipedia.org/wiki/Thunk_(functional_programming))-returning functions ([continuation-passing style](https://en.wikipedia.org/wiki/Continuation-passing_style)).
```

Comment:
To be clear, you're _not_ saying, "don't call it a trampoline", yes?  There is [precedent](https://www.npmjs.com/search?q=trampoline) in the JS ecosystem for this meaning. 

I would prefer we name it something silly, of course.

Open to suggestions.  I like `rainbow-trampoline`, because it supports functions of any color 🌈.

Reply:
Function coloring is also a loaded term. Sync/async coloring is currently the main kind in JS, but coloring has been used in the context of a couple other proposals lately.

Reply:
I can, without rancor, use the more-widely-understood meaning of words within their context (in this case, the community of JavaScript developers) while disregarding those less-widely-understood meanings. Naming is hard and we do the best we can 😄 

But if you (or anyone else) has a suggestion for naming this package, please offer it!

Reply:
`optional-a-sync`

Reply:
@mhofman @boneskull need more razors for this yak shaving?

I tend to use GPT for seeding name ideas. gpt-4o is really bad at it tho, had to go back to gpt-4
Here are some not awful seeds which we could use as inspiration for coming up with something reasonable.
```
1. AsyncBridge
2. SyncAsyncHarmonizer
3. JumpSync
4. YieldBounce
5. GenTrampolinator
6. AsyncOrNot
8. CodeChameleon
12. SyncAsyncLeap
16. AsyncFlip
17. YieldFlex
18. BounceBetween
19. SyncAsyncFusion
```

I still stand by `optional-a-sync` but these could also work from above:
`async-or-not` `sync-async-chameleon` `sync-async-bridge`

Reply:
I would like to name this `trampoline`.

Reply:
I'd name it "The Right Honourable Boris von Meija" if it were totally up to me, but I can live with `@endo/trampoline`.